subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/WatchableHierarchies.java

```diff
@@ -129,7 +145,7 @@ private SnapshotHierarchy invalidateUnsupportedFileSystems(SnapshotHierarchy roo
                     nonCombining()
                 );
         } catch (NativeException e) {
-            LOGGER.warn("Unable to list file systems to check whether they can be watched. The whole state of the virtual file system has been discarded. Reason: {}", e.getMessage());
+            LOGGER.warn("Unable to list file systems to check whether they can be watched. The whole state of the virtual file system has been discarded.", e);
```

Comment:
Why change this?

Reply:
I thought it would make sense to rely on the logger for deciding what to print of the exception. So with stacktrace or without. Do you have a strong opinion either way?

Reply:
My understanding is that we did not want to print the stacktrace, since for a `NativeException` there's rarely anything useful there. If you think there might be, I think it's fine to make the change.

Reply:
I went back to only printing the message.